include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -74,23 +71,12 @@ namespace aspect
             Point<3>
             push_forward(const Point<3> &chart_point) const;
 
-            Point<dim>
-            rotate_rectangle_to_parallelogram(const Point<dim> &x) const;
-
-            Point<dim>
-            rotate_parallelogram(const Point<dim> &x) const;
-
-
-
           private:
             double semi_major_axis_a;
             double eccentricity;
             double semi_minor_axis_b;
-            double rot_para_to_para_angle;
-            double para_to_rect_angle;
-            double rotation_longitude;
-            double rotation_latitude;
             double bottom_depth;
+            std::vector<std::vector<double> > corners;
```

Comment:
The documentation is added now. About the type, I remember that there where some conversion problems internally. But I will look into that again.